src/SplitPackages/Program.cs

```diff
@@ -331,5 +505,98 @@ private class Arguments
             public string CsvFile { get; set; }
             public string DestinationFolder { get; set; }
         }
+
+        private class ProjectJsonFileBuilder
+        {
+            private readonly string _path;
+            private readonly bool _whatIf;
+            private readonly bool _warningsAsErrors;
+            private readonly ILogger _logger;
+
+            public ProjectJsonFileBuilder(
+                string path,
+                bool whatIf,
+                bool warningsAsErrors,
+                ILogger logger)
+            {
+                _path = path;
+                _whatIf = whatIf;
+                _warningsAsErrors = warningsAsErrors;
+                _logger = logger;
+            }
+
+            private IList<PackageItem> _dependencies;
+            private IList<string> _frameworks = new List<string>();
+
+            public void AddDependencies(IList<PackageItem> dependencies)
+            {
+                _dependencies = dependencies;
+            }
+
+            public void AddFramework(Framework fx)
+            {
+                switch (fx)
+                {
+                    case Framework.Net451:
+                        _frameworks.Add("net451");
+                        break;
+                    default:
+                        break;
```

Comment:
Probably, I'm not sure if adding more TFMs will affect restore using this project.json so I would just leave it like this for now, it's not crazy complex.